circle.yml

```diff
@@ -1,4 +1,6 @@
 machine:
+  pre:
+    - curl -sSL https://s3.amazonaws.com/circle-downloads/install-circleci-docker.sh | bash -s -- 1.10.0
```

Comment:
sth weird happened with docker. i thought i had removed 1.9 and caching worked, but not after it got merged. we still have to revisit the cache situation. from a time perspective restoring and saving the cache takes almost as time as building it.